cha-ching-app/src/App.js

```diff
@@ -10,32 +10,30 @@ function App() {
 
   const isEmpty = (object) => Object.keys(object).length === 0;
 
-  function getClientSecret() {
+  useEffect(() => {
```

Comment:
This is something we can test for the next PR.

Reply:
I agree! lets add lots of tests for our app page!